src/app/common/templates/job-body.html

```diff
@@ -1,6 +1,13 @@
 <div class="job-body-heading">
     <div>
-        <div class="job-title">{{ job.attributes.name }}</div>
+        <div class="job-title">
+            <div ng-if="!(job.attributes.name_translation.from !== job.attributes.name_translation.to && dt.job_item)">
+                {{ job.attributes.name }}
+            </div>
+            <div ng-if="job.attributes.name_translation.from !== job.attributes.name_translation.to && dt.job_item">
```

Comment:
Not with the current architecture. I have to rewrite the job-body directive (which actually is not used like a directive but as a partial) and if we rewrite only this part of code it will become inconsistent with all the other code in the project.